templates/web/fixmystreet.com/footer_extra.html

```diff
@@ -31,7 +31,7 @@ <h2 class="mysoc-footer__site-name">FixMyStreet</h2>
                             %]>[% loc("Local alerts") %]</[% c.req.uri.path == '/alert' ? 'span' : 'a' %]></li>
                     </ul>
                     <ul>
-                        <li><a href="https://www.fixmystreet.com/council">FixMyStreet for Councils</a></li>
+                        <li><a href="/council">FixMyStreet for Councils</a></li>
```

Comment:
Could this (and the others) be to `/about/council`, as that's where the `/council` redirect takes you?